keras/objectives.py

```diff
@@ -32,8 +32,8 @@ def squared_hinge(y_true, y_pred):
 
 def hinge(y_true, y_pred):
     return T.maximum(1. - y_true * y_pred, 0.).mean(axis=-1)
-
-
+    
+    
```

Comment:
Also, please no style changes, such as deletion of empty lines or insertion of tabs.